docs/source/index.mdx

```diff
@@ -21,30 +21,17 @@ limitations under the License.
 
-[OpenVINO](https://docs.openvino.ai/latest/index.html) is an open-source toolkit that enables high performance inference capabilities for Intel CPUs, GPUs, and special DL inference accelerators. It is supplied with a set of tools to optimize and quantize models. Optimum Intel provides a simple interface to optimize Transformer models, convert them to OpenVINO Intermediate Representation format and to run inference using OpenVINO.
-
-## Installation
-
-To install the latest release of 🤗 Optimum Intel with the corresponding required dependencies, you can do respectively:
-
-| Accelerator                                                                                                            | Installation                                        |
-|:-----------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------|
-| [Intel Neural Compressor (INC)](https://www.intel.com/content/www/us/en/developer/tools/oneapi/neural-compressor.html) | `python -m pip install "optimum[neural-compressor]"`|
-| [Intel OpenVINO](https://docs.openvino.ai/latest/index.html)                                                           | `python -m pip install "optimum[openvino,nncf]"`    |
-
-We recommend creating a [virtual environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment) and upgrading
-pip with `python -m pip install --upgrade pip`.
-
-Optimum Intel is a fast-moving project, and you may want to install from source with the following command:
-
-```bash
-python -m pip install git+https://github.com/huggingface/optimum-intel.git
-```
-
-or to install from source including dependencies:
-
-```bash
-python -m pip install "optimum-intel[extras]"@git+https://github.com/huggingface/optimum-intel.git
-```
-
-where `extras` can be one or more of `neural-compressor`, `openvino`, `nncf`.
+[OpenVINO](https://docs.openvino.ai/latest/index.html) is an open-source toolkit that enables high performance inference capabilities on a variety of Intel [processors](https://docs.openvino.ai/latest/openvino_docs_OV_UG_supported_plugins_Supported_Devices.html). It is supplied with a set of tools to optimize your models with compression techniques such as quantization, pruning and knowledge distillation. Optimum Intel provides a simple interface to optimize your Transformers and Diffusers models, convert them to the OpenVINO Intermediate Representation (IR) format and run inference using OpenVINO Runtime.
```

Comment:
I like having, CPUs, GPUs and DL accelerators spelled out. Could we keep that?

Reply:
Will add it back !